### DISCUSSION TITLE
Errors with my profile

### DISCUSSION THREAD

**Discussion by JorgeAJT** [community/community#110489] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I just started to use Github a month ago. I write here because I don't know why, my last activities on Github cannot be seen in my profile, previously yes, but I don't know why not. I did some commits of any of my repositories but in my profile it's like I didn't anything, I attach screenshots.

![contributions](https://github.com/community/community/assets/154094092/70a13ac2-3c25-4c36-9f21-bce9e95d1319)

![repositories with last commits](https://github.com/community/community/assets/154094092/a45dccad-c3d3-449d-8006-7746c5fd0799)

Other thing is that the part of "Achievements". I don't have the achievement of to do my first merged, but in fact I did, and some friends have this but I don't know what's happening with my account/profile. The same I attach screenshots... 

![merged done](https://github.com/community/community/assets/154094092/92d4c502-4594-4fe7-bd83-5846b6a94d11)

Thanks

**Comment by DanielWren48** [comment]
Hi @JorgeAJT,
Your commits are not showing on your activity because you did those commits to a forked repository.
GitHub commits will only show up on your activity when you commit to your own repo and only to a main branch. 
Here is a reference to the official GitHub doc Read this [read it for better understanding](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile).

**Reply by JorgeAJT** [reply]
Hi, I normally create repos choosing my bootcamp academy as owner because I have limitless time to use Codespaces with them. And when I finish my work, I fork to have my own copy because with time the academy deletes it. But Doing like that, for example yesterday I did a commit in a repo which the academy is the owner and my profile showed me the green square of activity, so weird... 

Thanks for your answer @DanSmirnov48

**Comment by git-sumana** [comment]
I think you are adding your code as PR, rather than in your personal repo, commit in your main branch!
Check the path again!

**Reply by JorgeAJT** [reply]
I don't think so because I did only pull request once, I'm learning about git and github, so I don't think this is the problem. And at the momento I almost always work in the main branch, I only worked in other branch once, and I merged to the main branch but in my profile it doesn't appear the "Achievements" of merged.
